react/MuiCozyTheme/theme.js

```diff
@@ -83,16 +83,6 @@ export const normalTheme = createMuiTheme({
     borderRadius: defaultValues.borderRadius
   },
   breakpoints: {
-    // Define custom breakpoint values.
-    // These will apply to Material-UI components that use responsive
-    // breakpoints, such as `Grid` and `Hidden`. You can also use the
-    // theme breakpoint functions `up`, `down`, and `between` to create
-    // media queries for these breakpoints
-    // xs = all
-    // sm = tiny
-    // md = small
-    // lg = medium
-    // xl = large
```

Comment:
This is not a "theme exemple doc". It's in fact the correspondence table between UI's breakpoints and MUI's ones.